shopfloor_reception/services/reception.py

```diff
@@ -1247,14 +1352,20 @@ def process_with_new_pack(self, picking_id, selected_line_id, quantity):
                 picking, selected_line, message=message
             )
         response = self._set_quantity__process__set_qty_and_split(
-            picking, selected_line, quantity
+            picking,
+            selected_line,
+            quantity,
+            "process_with_new_pack",
+            is_over_reception_confirmed,
```

Comment:
```suggestion
            action="process_with_new_pack",
            is_over_reception_confirmed=is_over_reception_confirmed,
```
I'm not a fan of "hiding" kw args w/ positional args just because python allows to do so :)